src/gleam/option.gleam

```diff
@@ -312,6 +331,26 @@ pub fn lazy_or(first: Option(a), second: fn() -> Option(a)) -> Option(a) {
   }
 }
 
+fn reverse_acc(remaining, accumulator) {
```

Comment:
This looks like `reverse_list` to me

Reply:
Totally removed the reverse function here by implementing the `do_values` fucntion folding to the right.